cmd/wazero/wazero.go

```diff
@@ -133,6 +134,11 @@ func doRun(args []string, stdOut io.Writer, stdErr logging.Writer, exit func(cod
 			"This may be specified multiple times. When <wasm path> is unset, <path> is used. "+
 			"For read-only mounts, append the suffix ':ro'.")
 
+	var timeout time.Duration
+	flags.DurationVar(&timeout, "timeout", 0*time.Second,
+		"if a wasm binary runs longer than the given duration, then panic. "+
```

Comment:
for non gophers, the duration format for this string might not be common. Maybe add example here??

Reply:
the message is tad longer now, but it should be good (basically lifted the one in the Go docs for ParseDuration except the note about it being negative because it makes no sense)